java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_en_US.xml

```diff
@@ -5067,7 +5067,7 @@ organization. You may grant or remove the organization administrator role in the
         </context-group>
         </trans-unit>
         <trans-unit id="ssm.hw.systems.confirmmessage">
-          <source>{0} hardware {1} will be refreshed.</source>
+          <source>{0} hardware {1} &lt;a href="/rhn/schedule/PendingActions.do"&gt;{2} scheduled&lt;/a&gt; to be refreshed.</source>
```

Comment:
All the strings in this file are translated to various languages, particularly to Chinese by 3rd party translators. When we hide some words in the parameters (like "profile", "is/are") translators cannot figure it out and the translated text is most likely going to be broken.

So to fix this in a way that both supports translation and singular/plural distinction, I'd suggest we duplicate this entry to have a singular and a plural version. You can use `ssm.hw.systems.confirmmesage` as the singular version, and add another `<trans-unit>` entry with ID `ssm.hw.systems.confirmmessage.multiple` and write the plural text there. In the end, the text itself will be intact, and only parameter will be the number of profiles.